tests/hf/conftest.py

```diff
@@ -0,0 +1,67 @@
+import shutil
```

Comment:
Not sure if we want to keep this file or modify the tests to not depend on it.

Reply:
@soranjh up to you!